exhort/api/src/server.rs

```diff
@@ -162,9 +165,26 @@ async fn analyze(state: web::Data<AppState>, request: web::Json<AnalyzeRequest>)
     // mappings, go collect the vulnerability details from v11y, doing
     // our best effort and not allowing soft errors to fail the process.
     for vuln_id in vuln_ids {
-        match state.v11y_client.get_vulnerability(&vuln_id).await {
+        match state.v11y_client.get_cve(&vuln_id).await {
             Ok(vulnerabilities) => {
-                response.vulnerabilities.extend(vulnerabilities.iter().cloned());
+                if vulnerabilities.status() == 200 {
+                    let cve: Result<Box<RawValue>, _> = vulnerabilities.json().await;
+                    match cve {
+                        Ok(cve) => {
```

Comment:
```suggestion
                    match vulnerabilities.json::<Box<RawValue>>().await {
                        Ok(cve) => {
```
I don't know if it's better, but I don't love the shadowing of `cve`. Call it `raw` or something if you prefer the `let`.

Reply:
Oh, I love shadowing. Fisticuffs!

Reply:
whatevs. you asked for a review. :man_shrugging: